File: src/model/Order.java
package model;/*
author :Himal
version : 0.0.1
*/

import java.util.Date;

public class Order {
    private String orderId;
    private String orderCustomerId;
    private int orderNoOfItems;
    private Date orderDate;
    private Double orderAmount;
    private Double orderDiscountAmount;

    public Order() {
    }

    public Order(String orderId, String orderCustomerId, int orderNoOfItems, Date orderDate, Double orderAmount, Double orderDiscountAmount) {
        this.orderId = orderId;
        this.orderCustomerId = orderCustomerId;
        this.orderNoOfItems = orderNoOfItems;
        this.orderDate = orderDate;
        this.orderAmount = orderAmount;
        this.orderDiscountAmount = orderDiscountAmount;
    }

    public String getOrderId() {
        return orderId;
    }

    public void setOrderId(String orderId) {
        this.orderId = orderId;
    }

    public String getOrderCustomerId() {
        return orderCustomerId;
    }

    public void setOrderCustomerId(String orderCustomerId) {
        this.orderCustomerId = orderCustomerId;
    }

    public int getOrderNoOfItems() {
        return orderNoOfItems;
    }

    public void setOrderNoOfItems(int orderNoOfItems) {
        this.orderNoOfItems = orderNoOfItems;
    }

    public Date getOrderDate() {
        return orderDate;
    }

    public void setOrderDate(Date orderDate) {
        this.orderDate = orderDate;
    }

    public Double getOrderAmount() {
        return orderAmount;
    }

    public void setOrderAmount(Double orderAmount) {
        this.orderAmount = orderAmount;
    }

    public Double getOrderDiscountAmount() {
        return orderDiscountAmount;
    }

    public void setOrderDiscountAmount(Double orderDiscountAmount) {
        this.orderDiscountAmount = orderDiscountAmount;
    }
}

File: web/model/order.js
class  Order {
    constructor(orderId,orderCustomerId,orderNoOfItems,orderDate,orderAmount,orderDiscountAmount,orderItem) {
        this.orderId = orderId;
        this.orderCustomerId= orderCustomerId;
        this.orderNoOfItems = orderNoOfItems;
        this.orderDate = orderDate;
        this.orderAmount = orderAmount;
        this.orderDiscountAmount = orderDiscountAmount;
        this.orderItem = orderItem;
    }

    setOrderId =  function (orderId) {
        this.orderId = orderId;
    }

    getOrderId = function () {
        return this.orderId;
    }
    setOrderCustomerId = function (orderCustomerId) {
        this.orderCustomerId = orderCustomerId;
    }
    getOrderCustomerId = function () {
       return  this.orderCustomerId;
    }

    setOrderNoOfItems = function (orderNoOfItems) {
        this.orderNoOfItems = orderNoOfItems;
    }
    getOrderNoOfItems = function () {
        return this.orderNoOfItems;
    }

    setOrderDate = function (orderDate) {
        this.orderDate = orderDate;
    }
    getOrderDate = function () {
        return this.orderDate ;
    }

    setOrderAmount = function (orderAmount) {
        this.orderAmount = orderAmount;
    }
    getOrderAmount = function () {
        return this.orderAmount;
    }
    setOrderDiscountAmount = function (orderDiscountAmount) {
       this.orderDiscountAmount = orderDiscountAmount;
    }
    getOrderDiscountAmount = function () {
       return  this.orderDiscountAmount ;
    }

    setOrderItem = function (orderItem) {
        this.orderItem = orderItem;
    }
    getOrderItem = function () {
        return this.orderItem;
    }
}